app/helpers/medications_helper.rb

```diff
@@ -229,6 +143,50 @@ def days_checkbox
     end
   end
   # rubocop:enable MethodLength
-end
 
+  # rubocop:disable MethodLength, LineLength
+  def medication_unit_field(type)
+    {
+      id: "medication_#{type}_unit",
+      type: 'select',
+      name: "medication[#{type}_unit]",
+      dark: true,
+      value: @medication["#{type}_unit"] || t('medications.units.tablets.other'),
```

Comment:
I would just put `t('medications.units.tablets.other'),` on the next line and remove the disable LineLength comment.